config/env/development.js

```diff
@@ -11,18 +11,16 @@
  */
 
 module.exports = {
-  /***************************************************************************
-   * Set the default database connection for models in the development       *
-   * environment (see config/connections.js and config/models.js )           *
-   ***************************************************************************/
-  // models: {
-  //   connection: 'someMongodbServer'
-  // }
   log: {
     level: 'debug'
   },
+  // Disable default endpoints for each model e.g. GET /game/:id
   blueprints: {
     rest: false,
     shortcuts: false,
   },
+  // Needed to use custom 'select' statements using sails-disk
+  models: {
+    schema: true,
+  }
```

Comment:
Here are the [docs](https://sailsjs.com/documentation/reference/configuration/sails-config-models)

And here's the error that occurs when you query with `select` in development mode without this config:
```
"The provided criteria contains a custom `select` clause, but since this model (`user`) is `schema: false`, this cannot be relied upon... yet.  In the mean time, if you'd like to use a custom `select`, configure this model to `schema: true`. Or, better yet, since this is usually an app-wide setting,configure all of your models to have `schema: true` -- e.g. in `config/models.js`.  (Note that this WILL be supported in a future, minor version release of Sails/Waterline.  Want to lend a hand?  http://sailsjs.com/contribute)"
```